scripts/biomedical/chembl/README.md

```diff
@@ -0,0 +1,84 @@
+# Importing ChEMBL proteome data
+
+## Table of Contents
+
+- [Importing ChEMBL proteome data](#importing-chembl-proteome-data)
+  - [Table of Contents](#table-of-contents)
+  - [About the Dataset](#about-the-dataset)
+    - [Download URL](#download-url)
+    - [Overview](#overview)
+    - [Schema Overview](#schema-overview)
+    - [Notes and Caveats](#notes-and-caveats)
+    - [License](#license)
+  - [About the import](#about-the-import)
+  - [Examples](#examples)
+
+## About the Dataset
+
```

Comment:
Please include a description of the dataset here. It's ok to use an exact quote from the website - introduce what the data is.